frontend/src/components/pages/CampOverview/EditCamperModal/EditCamperModalForm.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+
+import {
+  HStack,
+  VStack,
+  Input,
+  Textarea,
+  FormControl,
+  FormLabel,
+  Text,
+} from "@chakra-ui/react";
+
+import {
+  EditCamperInfoFields,
+  EditModalSetterFunctions,
+} from "../../../../types/CamperTypes";
+import { FormQuestion } from "../../../../types/CampsTypes";
+import EditCamperFormResponseSection from "./EditCamperFormResponseSection";
+
+const EditCamperModalForm = ({
+  formQuestions,
+  formStateVariables,
+  setStateFuncs,
+}: {
+  formQuestions: FormQuestion[];
+  formStateVariables: EditCamperInfoFields;
+  setStateFuncs: EditModalSetterFunctions;
+}): JSX.Element => {
+  return (
+    <>
+      <HStack>
+        <VStack align="start" flexGrow="1">
+          <FormControl isRequired isInvalid={!formStateVariables.firstName}>
+            <FormLabel>
+              <Text fontSize="md" as="b">
+                First name
+              </Text>
+            </FormLabel>
+            <Input
+              onChange={(event) => {
+                setStateFuncs.setFirstName(event.target.value);
+              }}
+              value={formStateVariables.firstName}
+            />
+          </FormControl>
+        </VStack>
+        <VStack align="start" flexGrow="1">
+          <FormControl isRequired isInvalid={!formStateVariables.lastName}>
+            <FormLabel>
+              <Text fontSize="md" as="b">
+                Last name
+              </Text>
+            </FormLabel>
+            <Input
+              onChange={(event) => {
+                setStateFuncs.setLastName(event.target.value);
+              }}
+              value={formStateVariables.lastName}
+            />
+          </FormControl>
+        </VStack>
+        <VStack align="start" flexGrow="1">
+          <FormControl isRequired isInvalid={!formStateVariables.age}>
+            <FormLabel>
+              <Text fontSize="md" as="b">
+                Camper age
+              </Text>
+            </FormLabel>
+            <Input
+              onChange={(event) => {
+                setStateFuncs.setAge(parseInt(event.target.value, 10));
+              }}
+              value={formStateVariables.age}
+            />
+          </FormControl>
+        </VStack>
```

Comment:
These are difficult to read, can we create a component for the `VStack` and children, and then pass in appropriate parameters for each response?

Reply:
done